src/anm/animation/element.js

```diff
@@ -2424,7 +2430,8 @@ Element.prototype.__adaptModTime = function(modifier, ltime) {
 Element.prototype.__pbefore = function(ctx, type) { };
 Element.prototype.__pafter = function(ctx, type) { };
 Element.prototype.filterEvent = function(type, evt) {
-    if (this.shown) this.__saveEvt(type, evt);
+    /*if (this.visible && this.active)*/
+    this.__saveEvt(type, evt);
```

Comment:
For the moment, cached events (like mouse events) are handled strictly before the tweens on the next frame and then cache is cleared. This technique probably outdated and may be removed.